networking_generic_switch/devices/corsa_devices/corsavfc.py

```diff
@@ -450,6 +467,22 @@ def get_bridge(headers,
     return r
 
 
+#
+# GET INFO
+#
+#   200
+#   403 Forbidden
+def get_info(headers,
+             url_switch,
+             info_url):
+
+    try:
+        r = requests.get(info_url, headers=headers, verify=False)
+    except Exception as e:
```

Comment:
This `except` block can be removed if it is not doing anything but re-raising.

Reply:
This is a utility file that interacts directly with the switch.  There is a lot more functionality that we might need to add here as we add features. 

I think these functions need to be filled out to check the returned error status (like the others) but we haven't done that yet.